scripts/apiCall.js

```diff
@@ -0,0 +1,30 @@
+const https = require('https');
+const fs = require('fs');
+
+var args = process.argv;
+var options = {
+  hostname: args[2],
+  port: args[3],
+  path: args[4],
+  method: 'POST',
+  key: fs.readFileSync('certs/server/my-server.key.pem'),
+  cert: fs.readFileSync('certs/server/my-server.crt.pem'),
+  rejectUnauthorized: false
```

Comment:
You can get rid of this, I think. https://knilxof.org:5300/ now uses a valid cert.